website/docs/gator.md

```diff
@@ -0,0 +1,180 @@
+---
+id: gator
+title: The gator CLI
+---
+
+The `gator` CLI is a tool for testing Gatekeeper ConstraintTemplates and
+Constraints on Kubernetes objects.
+
+We plan on adding more subcommands in the future.
+
+## Installation
+
+To install `gator`, you may either
+[download the binary](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.7.0)
```

Comment:
should we point to `/releases` instead and mention that the feature is supported in version v3.7.0+ releases? this way we dont have to keep updating this doc.

Reply:
Good point, done!